jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/JibPluginTest.java

```diff
@@ -212,11 +143,19 @@ public void testWebAppProject() {
     Task warTask = tasks.getByPath(":war");
     Assert.assertNotNull(warTask);
 
+    Task jibDependenciesTask = tasks.getByPath(":jibDependencies");
+    Set<Task> taskDependencies = jibDependenciesTask.getDependsOn()
+        .stream()
+        .filter(it -> it instanceof TaskProvider)
+        .map(it -> ((TaskProvider<?>) it).get())
+        .collect(Collectors.toSet());
+
+    Assert.assertTrue(taskDependencies.contains(warTask));
+
     for (String taskName : KNOWN_JIB_TASKS) {
-      List<TaskProvider<?>> taskProviders =
-          (List<TaskProvider<?>>) tasks.getByPath(taskName).getDependsOn().iterator().next();
-      Assert.assertEquals(1, taskProviders.size());
-      Assert.assertEquals(warTask, taskProviders.get(0).get());
+      TaskProvider<?> jibDependenciesTaskProvider =
+          (TaskProvider<?>) tasks.getByPath(taskName).getDependsOn().iterator().next();
+      Assert.assertEquals(jibDependenciesTaskProvider.get(), jibDependenciesTask);
```

Comment:
These test changes (and the ones below) are now checking for whether the `jibDependencies` task depends on the requisite tasks (like the `warTask` or `bootWarTask` below) and whether all the jib tasks depend on `jibDependencies` which is the only task they need to depend on now.